packages/@ember/-internals/views/lib/mixins/view_support.ts

```diff
@@ -170,7 +194,7 @@ let mixin = {
    @return {Ember.View} receiver
    @private
    */
-  appendTo(selector) {
+  appendTo(this: Component, selector: string | Element) {
```

Comment:
This should probably be a `SimpleElement` instead. We can use `Element` in the `hasDOM` block.